### ISSUE TITLE
`setTextVisibility` on `shaka.extern.TextDisplayer`

### ISSUE THREAD

**Issue by Cowboy-coder** [google/shaka-player#1452]
In the process of implementing a custom rendering of subtitles I'm implementing my own `TextDisplayer`. 

However, in the implementation of `setTextVisibility` am I supposed to disable the text track as well by calling `player.setTextTrackVisibility` or something? Cause I'm getting a bit confused regarding why this function exists at all on the interface. Related concern regarding `isTextVisible()`... I guess my point is why are these functions required by the interface when shaka-player already supports hiding and showing subtitles by default?

Cause to me `append`, `remove` and `destroy` should be enough for implementing the interface? And if I for whatever reason need to care about the visibility of the subtitles then I could just get that from the internal player state?

I might not have a full understanding of how this works yet so I'm sorry if this question is stupid!

**Comment by Cowboy-coder** [comment]
Thanks for that response. Makes stuffs a lot clearer!

I guess I got confused also because seems `player.getTextTracks()` always seems to return a `active == true` track. So I thought perhaps I'm responsible for deactivating that track, but then now I saw this [piece of code](https://github.com/google/shaka-player/blob/5bf66fdc01d459e80a1cda088c1a03d4e3acade6/lib/player.js#L1462-L1474). But I did notice like you said that if I turned off subtitles by calling `player.setTextTrackVisibility(false)` then they were no longer streamed.
